src/btree/bt_handle.c

```diff
@@ -524,9 +524,24 @@ __btree_conf(WT_SESSION_IMPL *session, WT_CKPT *ckpt)
 
     /* If this is the first time opening the tree this run. */
     if (F_ISSET(session, WT_SESSION_IMPORT) || ckpt->run_write_gen < conn->base_write_gen)
-        btree->base_write_gen = btree->run_write_gen = btree->write_gen;
+        btree->run_write_gen = btree->write_gen;
     else
-        btree->base_write_gen = btree->run_write_gen = ckpt->run_write_gen;
+        btree->run_write_gen = ckpt->run_write_gen;
+
+    /*
+     * Don't reset the transaction ids in recovery because rollback to stable in addition to stable
```

Comment:
Can we update the comment why we skip logged tables and metadata as RTS doesn't happen on them?

Reply:
Updated the comments to describe with reason to skip metadata and logged tables.